python/foglamp/services/core/scheduler/scheduler.py

```diff
@@ -1022,6 +1075,163 @@ def _schedule_row_to_schedule(cls,
             self._schedule_first_task(schedule_row, now)
             self._resume_check_schedules()
 
+    async def disable_schedule(self, schedule_id):
+        """
+        Find running Schedule, Terminate running process, Disable Schedule, Update database
+
+        :param schedule_id:
+        :return:
+        """
+        # Find running task for the schedule.
```

Comment:
can this method be broken down in smaller usable protected def?

Reply:
May be in a future sprint. This module is already huge and it will become a maintenance nightmare if increase the no. of methods.